timeless/mail/__init__.py

```diff
@@ -0,0 +1,4 @@
+""" MAIL module """
+from flask_mail import Mail
+
+MAIL = Mail()
```

Comment:
@ammaratef45 I'd like to disable it for development environment and tests. Let's make a puzzle for it.